vultr/terraform/cluster.tf

```diff
@@ -0,0 +1,24 @@
+resource "vultr_kubernetes" "this" {
+  region  = var.region
+  label   = var.label
+  version = var.k8s_version
+
+  node_pools {
+    node_quantity = 1
+    plan          = var.node_pools_plan
+    label         = var.label
+    auto_scaler   = var.auto_scaler
+    min_nodes     = var.min_node
+    max_nodes     = var.max_node
+  }
+}
+
+resource "vultr_kubernetes_node_pools" "this" {
```

Comment:
This node pool is duplicated. 

I get his error:

```
│ Error: error creating node pool: gave up after 4 attempts, last error: "{\"error\":\"VKE server error: Cannot create NodePool: Cannot add NodePool: cannot have duplicate node pool labels\",\"status\":500}"
```

Removing this works then.

Reply:
done